lib/inject.ex

```diff
@@ -8,21 +8,23 @@ defmodule Inject do
     :ok
   end
 
-  def i(mod) do
-    inject(mod)
-  end
-
-  def inject(mod) do
+  defmacro inject(mod) do
     if Application.get_env(:inject, :disabled) do
       mod
```

Comment:
Note that I didn't wrap this in a quote block. I can do it if needed, but I think it would be redundant since we're not modifying the incoming atom or AST (not that I would see inject/1 receiving an AST any time soon).

Reply:
Also, doing `quote(do: mod)` here throws warnings and I'm not sure why.... 🤔

Reply:
Since quote is kinda like literal copy paste, what would be the result of such code?